models/PostModel.js

```diff
@@ -8,7 +8,8 @@ const addPost = `
     "textContent",
     timestamp,
     "isRepost",
-    "isQuotePost";
+    "isQuotePost",
+    "userId";
```

Comment:
The reason the user id is required on the frontend is for the validation of whether the author of the post is the active user. Without this return of the user id, the conditional renders inside the post menu do not work on any newly added posts or replies because we are not able to check the author id of the new post.

Reply:
Ohh I see that makes sense, do you think we should we rename this `authorId` then?

Reply:
yeah sure, just to confirm, you want to update the row name to authorId instead of userId for the post table right?

Reply:
As discussed offline, we can leave it as is. I thought this was a variable we made, so I thought it'd be easy to change, but I don't really wanna change the database schema lol